Makefile

```diff
@@ -131,6 +109,12 @@ cmdexecutor:
 	@echo "Building command executor binary"
 	@cd cmd/cmdexecutor && go build $(BUILD_OPTIONS) -o $(BIN)/cmdexecutor
 
+storkctl:
```

Comment:
It would be nice to dockerize storkctl to that people can run it using `kubectl run`.  https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#run

We can give the stork service account to the `kubectl run` command so it has access to all relevant resources.

Reply:
I'm going to put the linux binary in the container, so that users can use `kubectl exec` in the stork pods. Using `kubectl run` sounds like an anti-pattern though, if you can assign accounts to pods might as well run it directly.